.github/workflows/snl-ci.yml

```diff
@@ -53,3 +53,38 @@ jobs:
         working-directory: kokkos/build
         run: ctest --output-on-failure --timeout 3600
 
+  HIP_5_6_1:
+    name: SNL_HIP_ROCM_5_6_1
+    runs-on: [snl-kk-env-openblas-0.3.23-hip-5.6.1-latest]
+
+    steps:
+      - name: checkout_kokkos
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          repository: kokkos/kokkos
+          path: kokkos
```

Comment:
This is not critical but also note that the `repository` field is unnecessary.
So is path, technically, and we are not super consistent, in the next step we do `cd` and in later ones we use the `working-directory` directive.
(not blocking)